### PR TITLE
Demo for custom column renderer

### PR DESCRIPTION
**Pull request by Toshegg**
Connected to #23

### CODE REVIEW

File: demo/px-data-grid-custom-renderer-demo.html [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+<!-- Demo DOM module -->
+<dom-module id="px-data-grid-custom-renderer-demo">
+  <template>
+    <custom-style>
+      <style include="px-demo-styles" is="custom-style"></style>
+    </custom-style>
+
+    <!-- Header -->
+    <px-demo-header
+        module-name='Custom grid renderer'
```
Comment (limonte) [inline demo/px-data-grid-custom-renderer-demo.html]: just 'custom rendered', the 'grid' word is misleading

File: demo/px-data-grid-custom-renderer-demo.html [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+        description="Grid supports custom data renderer, that can be passed to a column. Example can be found in source of this demo and in the sample renderers, e.g. px-data-grid-string-renderer."
```
Comment (limonte) [inline demo/px-data-grid-custom-renderer-demo.html]: "Grid" -> px-data-grid

File: demo/px-data-grid-custom-renderer-demo.html (hunk 1 above)
Comment (Toshegg) [reply demo/px-data-grid-custom-renderer-demo.html]: Done.

File: demo/px-data-grid-custom-renderer-demo.html (hunk 2 above)
Comment (Toshegg) [reply demo/px-data-grid-custom-renderer-demo.html]: Done.

**Pull request merged**